test/range_test.py

```diff
@@ -509,11 +509,13 @@ def output(self):
                         'spl': MockFile(base + '/something.spl'),
                     }
 
-        with self.assertRaises(NotImplementedError):
+        def test_raise_not_implemented():
```

Comment:
I'd suggest to add the following pattern to test/helpers.py:

```py
try:  # or ``if py_version < 2.7, import unittest2``
    import unittest2 as unittest
except ImportError:
    import unittest
```

then import `from helpers import unittest` in all test files.